src/client/components/NavigationBar/style.scss

```diff
@@ -0,0 +1,132 @@
+@import '../../style';
+
+$whitecolor: #fff;
+$fontcolor: #9b9b9b;
+
+.navigation-bar {
+  background: $whitecolor;
+  font-size: .8em;
+  height: 35em;
+  margin-left: 1em;
+  margin-top: 1em;
+  width: 10em;
+
+  div {
+    margin-left: 1em;
+  }
+
+  p {
+    margin-left: 2em;
+  }
+
+  .mainlinks {
+    margin-bottom: 1.9em;
+    padding-top: .6em;
+
+    p {
+      margin-top: 0;
+      text-align: left;
+    }
+  }
+
+  .shortcut {
+    margin-bottom: 1.9em;
+
+    p {
+      color: $fontcolor;
+      margin-top: 0;
+      text-align: left;
+    }
+  }
+
+  .explore {
+    margin-bottom: 1.9em;
+
+    p {
+      color: $fontcolor;
+      margin-top: 0;
+      text-align: left;
+    }
+  }
+
+  h1 {
+    font-size: 1em;
+    font-weight: normal;
+    margin-left: .5em;
+    padding-left: .2em;
+    text-transform: uppercase;
+  }
+
+  .friends {
+    padding-bottom: .9em;
+
+    p {
+      margin-top: -.2em;
+    }
+  }
+
+  .games {
+    padding-bottom: .9em;
+
+    p {
+      margin-top: -.2em;
+    }
+  }
+
+  button {
```

Comment:
if you write so many settings on an element you should step back and think through the whole thing. You dont need these settings.